app/controllers/users.js

```diff
@@ -107,46 +112,31 @@ exports.create = function(req, res) {
   }
 };
 
-/**
- * Assign avatar to user
- */
-exports.avatars = function(req, res) {
-  // Update the current user's profile to include the avatar choice they've made
-  if (req.user && req.user._id && req.body.avatar !== undefined &&
-    /\d/.test(req.body.avatar) && avatars[req.body.avatar]) {
-    User.findOne({
-      _id: req.user._id
-    })
-    .exec(function(err, user) {
-      user.avatar = avatars[req.body.avatar];
-      user.save();
-    });
-  }
-  return res.redirect('/#!/app');
-};
 
-exports.addDonation = function(req, res) {
-  if (req.body && req.user && req.user._id) {
+exports.addDonation = function (req, res) {
+  if (req.body && req.user && req.user._id) { //eslint-disable-line
     // Verify that the object contains crowdrise data
-    if (req.body.amount && req.body.crowdrise_donation_id && req.body.donor_name) {
+    if (req.body.amount && req.body.crowdrise_donation_id
+       && req.body.donor_name) {
       User.findOne({
-        _id: req.user._id
+        _id: req.user._id //eslint-disable-line
       })
-      .exec(function(err, user) {
+        .exec((err, user) => {
         // Confirm that this object hasn't already been entered
-        var duplicate = false;
-        for (var i = 0; i < user.donations.length; i++ ) {
-          if (user.donations[i].crowdrise_donation_id === req.body.crowdrise_donation_id) {
-            duplicate = true;
+          let duplicate = false;
+          for (let i = 0; i < user.donations.length; i++) { //eslint-disable-line
+            if (user.donations[i].crowdrise_donation_id
+              === req.body.crowdrise_donation_id) {
+              duplicate = true;
+            }
+          }
+          if (!duplicate) {
+            console.log('Validated donation');
```

Comment:
Do we really need this `console.log('Validated donation');` here? If this was used during development then you can take this.

Reply:
The console.log() statements was used for development and I deleted the user test. I will fix that right away.